site/pages/docs/specification.mdx

```diff
@@ -50,7 +50,7 @@ A Mini App URL must have a FrameEmbed in a serialized form in the `fc:frame` met
 
 | Property              | Type   | Required | Description                              | Constraints                                  |
 |-----------------------|--------|----------|------------------------------------------|----------------------------------------------|
-| type                  | string | Yes      | Mini App name.                           | Max length 32 characters                     |
+| type                  | string | Yes      | One of: `launch_frame`, `view_token`           |                      |
```

Comment:
1. Can you realign the closing `|` to line up with the others, like it was before?
2. Realizing we should put the "One of: `launch_frame`, `view_token`" under `Constraints`, not `Description`
3. As for `Description`, let's update it to something correct, eg. "The type of action"